metadata.yaml

```diff
@@ -11,3 +11,6 @@ releaseSeries:
   - major: 0
     minor: 3
     contract: v1alpha3
+  - major: 1
```

Comment:
what's the ordering here? should it be at the top instead (newest first)?

Reply:
Good point, not sure 😄 — I'll have it at the top

Reply:
FYI: I believe the order does not matter a lot. Even if the contract version matches clusterctl picks up the entry with the newest release series. 
That said it is good to have it at the top. 🙂